native/src/routes/Pois.tsx

```diff
@@ -74,6 +74,7 @@ const Pois = ({ pois: allPois, language, cityModel, route, navigation }: PoisPro
   const { slug, multipoi } = route.params
   const [sheetSnapPointIndex, setSheetSnapPointIndex] = useState<number>(1)
   const [listScrollPosition, setListScrollPosition] = useState<number>(0)
+  const [deselectOnBackNavigation, setDeselectOnBackNavigation] = useState(slug === undefined)
```

Comment:
I think this should be
```suggestion
  const [deselectOnBackNavigation, setDeselectOnBackNavigation] = useState(slug === undefined && multipoi === undefined)
```